### ISSUE TITLE
Used fonts are improperly discarded

### ISSUE THREAD

**Issue by GreenGremlin** [cssnano/cssnano#726]
Given the following css:

```css
@font-face {
    font-family: 'FontAwesome';
    src: url('/fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'),
         url('/fonts/fontawesome-webfont.woff?v=4.7.0') format('woff');
}

.icon {
    font: normal normal normal 14px/1 FontAwesome;
}
```

...the FontAwesome @font-face definition is improperly removed. This appears to be [the expected behavior](https://github.com/cssnano/cssnano/blob/master/packages/postcss-discard-unused/src/__tests__/index.js#L88), however this is valid css and should properly be detected as a use of the font. Would you be open to a PR that adds support for proper font use detection with mixed quote styles?

**Comment by evilebottnawi** [comment]
@GreenGremlin i don't understand you, removing `@font-face` is unsafe operation so we don't do this. Please prove more information. What you actually have and what you expected.

**Comment by GreenGremlin** [comment]
I apologize, I should have mentioned that I'm using the postcss-discard-unused plugin specifically to remove unused css from a concatenated css file. I believe this is also possible through cssnano using the `discardUnused` option. The problem is that, in this case, it is removing a font that is actually used.

**Comment by evilebottnawi** [comment]
@GreenGremlin can you provide example or minimum reproducible test repo?

**Comment by GreenGremlin** [comment]
I just created https://github.com/cssnano/cssnano/pull/727 to fix the issue. The code block above is a decent, minimal example. Simply processing that block of css through postcss with the postcss-discard-unused plugin enabled with the default settings will produce the bug.

```js
postcss([postcssDiscardUnused]).process(`
    @font-face {
        font-family: 'FontAwesome';
        src: url('/fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'),
             url('/fonts/fontawesome-webfont.woff?v=4.7.0') format('woff');
    }

    .fa {
        font: normal normal normal 14px/1 FontAwesome;
    }
`);
```

Should not discard the `fontAwesome` @font-face, but it does.

**Comment by smac89** [comment]
In case anyone stumbles upon this problem, may I suggest making sure that your fontawesome.css file does not have something like:

```css
i @font-face {
  font-family: 'FontAwesome';
  src: url('../../fonts/vendor/fontawesome-webfont.eot?v=4.1.0');
  src: url('../../fonts/vendor/fontawesome-webfont.eot?#iefix&v=4.1.0') format('embedded-opentype'), url('../../fonts/vendor/fontawesome-webfont.woff?v=4.1.0') format('woff'), url('../../fonts/vendor/fontawesome-webfont.ttf?v=4.1.0') format('truetype'), url('../../fonts/vendor/fontawesome-webfont.svg?v=4.1.0#fontawesomeregular') format('svg');
  font-weight: normal;
  font-style: normal;
}
```

I'm talking about the extra `i` prepended to the font declaration. After I removed the offending character, cssnano worked for me.

**Comment by ludofischer** [comment]
This seems fixed in 5.0.0-rc.2
